README.md

```diff
@@ -179,61 +153,55 @@ Armbian [releases](https://docs.armbian.com/Release_Changelog/) quarterly at the
+- The easiest way to help is by "Starring" our repository - it helps more people find our code.
 
-<p align=right><a href=#table-of-contents>⇧</a></p>
+- You also can maintain and develop [docs](https://github.com/armbian/documentation), [CI](https://github.com/armbian/ci-testing-tools), [autotests](https://github.com/armbian/autotests), [seed torrents](https://forum.armbian.com/topic/4198-seed-our-torrents/), help on [forum moderating](https://forum.armbian.com/topic/12631-help-on-forum-moderating/), [project administration](https://forum.armbian.com/forum/39-armbian-project-administration/), [costs](https://www.armbian.com/donate).
 
-## Contribute
+Please make sure to read the [Contributing Guide](.github/CONTRIBUTING.md) before you write any code.
 
-- Adding a new feature? 
+## Support
 
-    You are welcome to suggest or contribute directly to the code with a pull request. In case your proposed changes are large, remember to discuss them prior to development.
+- Community support
 
-- Join development? 
+    Armbian is free software and provides **best effort help** through [community forums](https://forum.armbian.com/). If you can't find answer there and/or with help of [general project search engine](https://www.armbian.com/search) and [documentation](https://docs.armbian.com), consider [hiring an expert](https://www.debian.org/consultants/).
 
-    Join regulars on their active missions, start maintaining any part of the code: patches, drivers or scripted applications like [armbian-config](https://github.com/armbian/config), address [community wishes](https://github.com/armbian/build/issues),
+- Personal support
 
-- Help elsewhere?
+    Personal support limited to active project supporters and sponsors. The shortest way to become one and receive our attention is a four figure [donation to our non-profit project](https://www.armbian.com/donate).
 
-    Maintain and develop [documentation](https://github.com/armbian/documentation), [CI](https://github.com/armbian/ci-testing-tools), [autotests](https://github.com/armbian/autotests), [seed torrents](https://forum.armbian.com/topic/4198-seed-our-torrents/), help on [forum moderating](https://forum.armbian.com/topic/12631-help-on-forum-moderating/), [project administration](https://forum.armbian.com/forum/39-armbian-project-administration/), [costs](https://www.armbian.com/donate).
+## Contact
 
-<p align=right><a href=#table-of-contents>⇧</a></p>
+- [Forums](https://forum.armbian.com) for Participate in Armbian
+- IRC: `#armbian` on Libera.chat
+- Follow [@armbian](https://twitter.com/armbian) on twitter or [LinkedIn](https://www.linkedin.com/company/armbian).
+- Bugs: [issues](https://github.com/armbian/build/issues) / [JIRA](https://armbian.atlassian.net/jira/software/c/projects/AR/issues)
```

Comment:
Replace it with this link - its much better organised:
https://armbian.atlassian.net/jira/dashboards/10000